tests/Microsoft.Identity.Test.Integration.Win8/Microsoft.Identity.Test.Integration.Win8.csproj

```diff
@@ -19,7 +19,7 @@
       <Name>Microsoft.Identity.Test.LabInfrastructure</Name>
     </ProjectReference>
     <ProjectReference Include="..\Microsoft.Identity.Test.Common\Microsoft.Identity.Test.Common.csproj" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.9.4" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.2.0" />
```

Comment:
This is a false positive, because `Microsoft.NET.Test.Sdk` UAP version references affected netwonsoft. We don't use UAP version.

Still, resolution lies with affected product.